functions/Install-DbaMaintenanceSolution.ps1

```diff
@@ -315,7 +320,7 @@ function Install-DbaMaintenanceSolution {
                 $required = @('CommandExecute.sql')
             }
 
-            if ($LogToTable) {
+            if ($LogToTable -and $InstallJobs -eq $false) {
```

Comment:
Also - not following this add, wouldn't we want the CommandLog table to be created whether jobs are installed or not? (or did I miss that it's added in later if InstallJobs is true?

Reply:
If -InstallJobs:$true then the only script which can be run is the "MaintenanceSolution.sql" script which will force install the commandlog table anyway.

The add was done to make it the decision between "Install" and "Config" and to while the maintenancesolution will skip over the object already being present with the existence check, to stop unnecessary scripts from being run.

I walked through the logic with @niphlod over Slack and advised this is the route from #6704 that was envisaged

Reply:
Cool gotcha - thanks for clarrifying